staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -315,9 +323,32 @@ func cleanVerb(verb string, request *http.Request) string {
 	if verb == "WATCHLIST" {
 		reportedVerb = "WATCH"
 	}
+	if verb == "PATCH" && request.Header.Get("Content-Type") == string(types.ApplyPatchType) {
+		reportedVerb = "APPLY"
+	}
 	return reportedVerb
 }
 
+func cleanDryRun(dryRun []string) string {
+	if errs := validation.ValidateDryRun(nil, dryRun); len(errs) > 0 {
+		return "invalid"
+	}
+
```

Comment:
`strings.Join(sets.NewString(dryRun...).List(),",")` will dedupe and sort for you if you want to collapse onto that